### PR TITLE
Gridsome

### PR DESCRIPTION
**Pull request by rahmot**
This is gridsome project folder to help us start with full implementation of GMN site

### CODE REVIEW

**Review by bebatut — COMMENTED** [review]
Thanks. Before I can really try locally, can you
- remove all files that do not belong this PR?
- move the content of the `my-gridsome-site` folder directly there?
- add in the README file the information to build the website locally?

**Comment by rahmot** [comment]
@bebatut I have made changes to the PR and the gridsome folder. How do you see that?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,47 @@
-Galaxy Mentor Network 
+Galaxy Mentor Network
 =====================
+
+# How can I generate the website locally?
+
+### 1. Install Gridsome CLI tool if you don't have
```
Comment (bebatut) [inline README.md:6]: It may nice to have instructions as in the Hub: https://github.com/galaxyproject/galaxy-hub/#getting-started

**Comment by rahmot** [comment]
> I managed to run the website following the instructions I put in the README file

Thanks so much for doing this. I planned to do it after creating the `CONTRIBUTING.md`. The instructions are explicit. Merci :)

**Comment by bebatut** [comment]
I merge that PR so the structure of the website with Gridsome is there

**Pull request merged**